pkgs/development/python-modules/bx-python/default.nix

```diff
@@ -3,14 +3,14 @@
 
 buildPythonPackage rec {
   pname = "bx-python";
-  version = "0.8.11";
+  version = "0.8.12";
   disabled = isPy27;
```

Comment:
Could be changed to `pythonOlder "3.6"` now that bx-python has dropped Python 3.5 support.

Reply:
Great catch, thanks!